service/installer/helm/helm.go

```diff
@@ -32,8 +32,8 @@ var HelmInstallerType spec.InstallerType = "HelmInstaller"
 // Config represents the configuration used to create a Helm Installer.
 type Config struct {
 	// Dependencies.
-	Configurer configurerspec.Configurer
-	Logger     micrologger.Logger
+	Configurers []configurerspec.Configurer
```

Comment:
Being able to configure multiple configurers allows us to add for instance a secret configurer next to the configmap configurer. This means we can merge the values provided by draughtsman configmap and secret with Helm.